lib/core/tests/bench_test.js

```diff
@@ -0,0 +1,6 @@
+export function set_gradient_bg(dom, r, g, b) {
+  let components = r + "," + g + "," + b;
```

Comment:
Or `[r,g,b].join()` if you like it

Reply:
Wouldn't the explicit string concatenation be faster? I am calling this function 400 times per frame.